plugins/modules/ipam_address_block.py

```diff
@@ -2399,6 +2412,9 @@ def find(self):
                     return None
                 raise e
         else:
+            # If address is None, return None, indicating next_available_address block should be created and not updated
```

Comment:
```suggestion
            # If address is None, return None, indicating next_available_address block should be created
```

If "not updated" here refers to an address , then the filter is used not only for updation but creation of the address too , hence "not updated" is invalid

Reply:
Modified!